services/simcore/docker-compose.yml.j2

```diff
@@ -92,7 +92,7 @@ services:
         # NOTE: apiserver does not need sslheader since there is no socket.io
         - traefik.http.routers.${SWARM_STACK_NAME}_api-server.middlewares=${SWARM_STACK_NAME}_gzip@swarm
         - traefik.http.routers.${SWARM_STACK_NAME}_apiserver_swagger.service=${SWARM_STACK_NAME}_api-server
-        - traefik.http.routers.${SWARM_STACK_NAME}_apiserver_swagger.rule=PathPrefix(`/dev/`)
+        - traefik.http.routers.${SWARM_STACK_NAME}_apiserver_swagger.rule=PathPrefix(`/dev/`) || PathPrefix(`/doc/`)
```

Comment:
Q: is `/doc/` the same as `/doc`? If not, then, as I mentioned in the [issue](https://github.com/ITISFoundation/osparc-ops-environments/issues/1033) i would like to have the latter